task-manager-backend/src/task/controllers/delete-task/delete-task.js

```diff
@@ -0,0 +1,24 @@
+const ApiResponse = require("../../../utils/api-response");
+const ApiError = require("../../../utils/api-error");
+const { deleteTaskUsecase } = require("../../usecases");
+
+const { TASK_DELETE_SUCCESS } = require("../../../commons/constants");
+
+module.exports = {
+    deleteTask: {
+        path: "/:id",
```

Comment:
this should be task/:taskId
it should be like what u are deleting(task) and then id of that task

Reply:
in app, '/tasks' is already defined.

If I add whole path in each file, then I need to remove parent path from app.js.

Did I need to change it?